src/raw_node.rs

```diff
@@ -410,6 +410,20 @@ impl<T: Storage> RawNode<T> {
         Err(Error::StepPeerNotFound)
     }
 
+    /// A callback when entries are fetched asynchronously .
```

Comment:
Should mention the `context` parameter is the one pass to `Storage::entries` previously, and how this function may panic.